src/functions-and-arrays.js

```diff
@@ -1,20 +1,54 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  }
+  if (a < b) {
+    return b;
+  }
+  if (a = b) {
+    return a
+  }
+}
+//console.log(maxOfTwoNumbers(2, 1));
+//console.log(maxOfTwoNumbers(5, -7));
+//console.log(maxOfTwoNumbers(1, 3));
+//console.log(maxOfTwoNumbers(-5, 3));
+//console.log(maxOfTwoNumbers(4, 4));
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
+  if (arr.length === 1) {
+    return arr[0];
+  }
+  let longestWord = "";
+  //iterate over each words in the array
+  for (let i = 0; i < arr.length; i++) {
+    if (longestWord.length < arr[i].length) {
+      longestWord = arr[i];
+    }
+  }
```

Comment:
I think you missed to `return longestWord` here. This is why the last 2 tests are failing.